src/js/resize-manager.js

```diff
@@ -71,7 +71,18 @@ class ResizeManager extends Component {
           return;
         }
 
-        Events.on(this.el_.contentWindow, 'resize', this.debouncedHandler_);
+        const debouncedHandler_ = this.debouncedHandler_;
+        let unloadListener_ = this.unloadListener_ = function() {
+          Events.off(this, 'resize', debouncedHandler_);
+          Events.off(this, 'unload', unloadListener_);
+
+          unloadListener_ = null;
+        };
+
+        // safari and edge can unload the iframe before resizemanager dispose
+        // we have to dispose of event handlers correctly before that happens
+        Events.on(this.el_.contentWindow, 'unload', unloadListener_);
```

Comment:
We have to listen to the content window `unload` event so that we always dispose the iframe correctly if we add events to it. We also make sure to use the `context` of the `contentWindow` as `this` since IE11 does not work any other way for some reason.